src/BTVNanoCommissioning/workflows/ctag_eWc_valid_sf.py

```diff
@@ -484,9 +474,10 @@ def process_shift(self, events, shift_name):
         if self.isArray:
             # Keep the structure of events and pruned the object size
             pruned_ev = events[event_level]
-            pruned_ev.Jet = sjets
-            pruned_ev.Muon = shmu
+            pruned_ev["Jet"] = sjets
+            pruned_ev["Muon"] = shmu
             pruned_ev["MuonJet"] = smuon_jet
+            pruned_ev["OtherJets"] = sotherjets
```

Comment:
Do you still want to store the jet flag instead of collections on MuonJet & OtherJet? The `mujetsel` can still considered as a flag in the Jet variables